lib/ansible/cli/galaxy.py

```diff
@@ -1218,6 +1218,9 @@ def execute_init(self):
                     with open(dest_file, 'wb') as df:
                         df.write(b_rendered)
                 else:
+                    if f + '.j2' in files:
+                        # We have two possible sources for the same file, we prefer the jinja template
+                        continue
```

Comment:
This shouldn't apply in the templates dir since .j2 files are not templated. Can you add a test case for templates/?

Reply:
I made tests for this case and made the tests pass.
CI failed though. At least one of the failed tests looks like a network problem.

Reply:
Thanks :+1:

It looks like an extra commit made it onto this branch that has already been merged to devel. If you rebase to remove the extra change https://github.com/ansible/ansible/pull/82624/commits/aca94cdc1ecbc746f95de2686ac19877fd7c656d, that will also rerun the tests.